dlc-wallet-test/src/test/scala/org/bitcoins/dlc/wallet/DLCExecutionTest.scala

```diff
@@ -6,10 +6,10 @@ import org.bitcoins.commons.jsonmodels.dlc.DLCMessage.{
   SingleNonceContractInfo
 }
 import org.bitcoins.commons.jsonmodels.dlc.DLCState
-import org.bitcoins.commons.jsonmodels.dlc.DLCStatus.{
-  Claimed,
-  Refunded,
-  RemoteClaimed
+import org.bitcoins.commons.jsonmodels.dlc.SerializedDLCStatus.{
```

Comment:
Didn't this get renamed?

Reply:
Yup, this is causing CI failure: https://travis-ci.org/github/bitcoin-s/bitcoin-s/jobs/747088876#L472